Source/CombatExtended/CombatExtended/ArmorUtilityCE.cs

```diff
@@ -235,7 +235,11 @@ private static bool TryPenetrateArmor(DamageDef def, float armorAmount, ref floa
                 else
                 {
                     // Hard armor takes damage as reduced by damage resistance and can be almost impervious to low-penetration attacks
-                    var armorDamage = Mathf.Max(1, newDmgAmount);
+                    var fullDeflect = armorAmount > penAmount * 2f;
+                    var armorDamagePen = penAmount * 2f - armorAmount;
+                    var armorDmgMult = fullDeflect ? 0 : penAmount == 0 ? 1 : Mathf.Clamp01(armorDamagePen / penAmount);
+                    var armorDmgAmount = dmgAmount * armorDmgMult;
+                    var armorDamage = Mathf.Max(1, armorDmgAmount);

```

Comment:
Since you've made it so armor would take damage even if the attack doesn't do any damage to the wearer, I don't think armor taking minimal damage of 1 every time it is hit is necessary anymore. As I said before, I don't think a squirrel should be able to break power armor with 500 hits.

Reply:
Done. To a degree I can kind of swing both ways on this argument. Squirrel is silly, but there's a pretty reasonable number of items that can probably absorb infinite punishment with this change.

Reply:
Also have you tested out the differences in effective damage to apparel, both early and lategame, and if so, what are your opinions on it?

Reply:
Can you make the armour taking minimum damage an optional toggle in the options menu?
Armour being completely immune to animals would actually make the game unplayable for me if this were to happen.